scripts/hyperkit.mjs

```diff
@@ -76,7 +137,7 @@ async function buildIfNotAccess(destPath, fn, mode = fs.constants.X_OK) {
   } catch (ex) {
     // The output must be rebuilt.
   }
-  const tmpDirPrefix = path.join(os.tmpdir(), `${ path.basename(destPath, '.exe') }-`);
+  const tmpDirPrefix = destPath.replace(/(?:\.exe)$/, '-');
```

Comment:
This is probably the 3rd time I see the stripping of the `.exe` extension to make a temp directory name. Maybe this should move into a separate function.

Or into a separate module uploaded to NPM? j/k 😄